From bbe9684426baafe920950691f86a7516a1ca4917 Mon Sep 17 00:00:00 2001
From: zachary <zhangzg@marvell.com>
Date: Thu, 9 Nov 2017 18:21:12 +0800
Subject: [PATCH 2074/2241] fix: dts: a3700: correct uboot env size in spi
 partition

There was an updating for uboot env store location. This patch
corrects uboot env spi partition size.

Change-Id: I3b74f66b9add7f40cd370eb2fc9e3609d334bdbb
Signed-off-by: zachary <zhangzg@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/46139
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db.dts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db.dts b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
index 12c523af..409c42d 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
@@ -81,15 +81,15 @@
 						#size-cells = <1>;
 						partition@0 {
 							label = "bootloader";
-							reg = <0x0 0x200000>;
+							reg = <0x0 0x3f0000>;
 						};
-						partition@200000 {
+						partition@3f0000 {
 							label = "U-boot Env";
-							reg = <0x200000 0x10000>;
+							reg = <0x3f0000 0x10000>;
 						};
-						partition@210000 {
+						partition@400000 {
 							label = "Linux";
-							reg = <0x210000 0xDF0000>;
+							reg = <0x400000 0xDF0000>;
 						};
 					};
 				};
-- 
2.7.4

